appspec.yml

```diff
@@ -1,13 +1,22 @@
 os: linux
+
 version: 0.0
+
 files:
   - source: app.js
     destination: /opt/unicorn
-
   - source: package.json
     destination: /opt/unicorn
+  - source: unicorn.service
+    destination: /opt/unicorn/scripts
```

Comment:
@aosyborg since amazon linux is forked from red hat, I don't believe it has systemd in /etc/